lib/iris/coords.py

```diff
@@ -1001,10 +1001,17 @@ def _guess_bounds(self, bound_position=0.5):
             raise ValueError('Coord already has bounds. Remove the bounds '
                              'before guessing new ones.')
 
-        diffs = np.diff(self.points)
-
-        diffs = np.insert(diffs, 0, diffs[0])
-        diffs = np.append(diffs, diffs[-1])
+        if getattr(self, 'circular', False) is True:
```

Comment:
Hi @cpelley. I'm :+1: overall, but would you mind dropping the `is True` (see PEP8)?